functions/fetch-yelp/index.js

```diff
@@ -2,6 +2,27 @@ const fetch = require('node-fetch');
 require('dotenv').config({ path: `.env.development.local` });
 
 const handler = async (event) => {
+  const { zip, search } = event.queryStringParameters;
+  try {
+    const resp = await fetch(`https://api.yelp.com/v3/businesses/search?location=${zip}&term=${search}`, {
+      headers: {
+        Authorization: `Bearer ${process.env.YELP_API_KEY}`,
+      },
+    });
+    const data = await resp.json();
+    const json = JSON.stringify(data);
```

Comment:
Here we can also use dot notation so that we don't have to remember to call .businesses in our fetch call.